sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -57,6 +57,8 @@
     private final boolean isAutoComplete;
     private final ReceiveMode receiveMode;
 
+    private static final DeadLetterOptions DEFAULT_DEAD_LETTER_OPTIONS =  new DeadLetterOptions();
+
```

Comment:
nit: group variables together by type. This is in between the private final variables. Generally I put static variables at the top arranged by visibility.